libvmi/os/windows/core.c

```diff
@@ -698,7 +701,30 @@ init_from_rekall_profile(vmi_instance_t vmi)
             windows->ntoskrnl_va = kpcr - kpcr_rva;
             if ( VMI_FAILURE == vmi_translate_kv2p(vmi, windows->ntoskrnl_va, &windows->ntoskrnl) )
                 goto done;
+		//Get kernel base address using cstar/lstar and KiSystemCall32Shadow / KiSystemCall64Shadow 
+        } else if ( VMI_SUCCESS == rekall_profile_symbol_to_rva(windows->rekall_profile, "KiSystemCall64Shadow", NULL, &int0_rva) ) {
 
+            vmi_get_vcpureg(vmi, &lstar, MSR_LSTAR, 0);
+            vmi_get_vcpureg(vmi, &cstar, MSR_CSTAR, 0);
+
+
+    		rekall_profile_symbol_to_rva(windows->rekall_profile, "KiSystemCall32Shadow", NULL, &kisystemcall32shadow);
+		    rekall_profile_symbol_to_rva(windows->rekall_profile, "KiSystemCall64Shadow", NULL, &kisystemcall64shadow);
```

Comment:
.. there is no point grabbing it here again.

Reply:
Youre absolutely right, i'm changing this